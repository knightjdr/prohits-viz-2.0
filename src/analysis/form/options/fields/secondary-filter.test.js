import React from 'react';
import { shallow } from 'enzyme';

import SecondaryFilter from './secondary-filter';

describe('SecondaryFilter', () => {
  test('Renders', () => {
    const wrapper = shallow(
      <SecondaryFilter
        analysisType="dotplot"
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
