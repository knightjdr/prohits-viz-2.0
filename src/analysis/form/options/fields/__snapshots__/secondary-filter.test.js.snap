// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryFilter Renders 1`] = `
<CustomField
  helpMessage="Interactions that do no pass the primary score filter but
    pass this secondary filter will be marked with an intermediate intensity
    edge in the dot plot. Interactions that do not pass either filter will be
    marked with a low intensity edge. The secondary filter can be adjusted
    depending on the dataset to allow a greater or lesser number of interactions
    into this 'medium' confidence range."
  inputType="number"
  label="Secondary filter"
  name="secondaryFilter"
  onChange={[Function]}
  placeHolder="Secondary filter..."
  type="input"
/>
`;
